### ISSUE TITLE
Figure 1 in higher resolution available?

### ISSUE THREAD

**Issue by BenK89** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#3562]
The quality of the image is very low... Do we have it in higher resolution?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 43c84773-337e-9df7-7fbd-fc9d94280711
* Version Independent ID: 223a78a0-67fd-8b96-be3f-153bb400101b
* Content: [Microsoft Teams call flows - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/microsoft-teams-online-call-flows#feedback)
* Content Source: [Teams/microsoft-teams-online-call-flows.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/microsoft-teams-online-call-flows.md)
* Service: **msteams**
* GitHub Login: @CarolynRowe
* Microsoft Alias: **crowe**

**Comment by scanum** [comment]
@BenK89 Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by TokyoScarab** [comment]
@BenK89 Thank you for raising this issue! I'll be looking into it!

**Comment by TokyoScarab** [comment]
@CarolynRowe I have made the replacement images that are requested in this issue but am unable to upload them to the media folder. Would you be able to assist in this?

**Comment by scanum** [comment]
@CarolynRowe Hi could you please assist with this issue? Thanks!

**Comment by scanum** [comment]
@CarolynRowe Hi could you please assist with this issue? Thanks!

**Comment by scanum** [comment]
@CarolynRowe Hi could you please assist with this issue? Thanks!

**Comment by CarolynRowe** [comment]
Can someone provide the updated graphics?

**Comment by TokyoScarab** [comment]
@CarolynRowe Yes, the first 2 images in the document have a much lower resolution than the rest of the images. I recreated them. Here are the links for them:
Image 1:  https://i.imgur.com/7u4OumQ.png
Image 2:  https://i.imgur.com/YPjwrMZ.png

**Comment by scanum** [comment]
@CarolynRowe Hope everything is going well. @TokyoScarab has provided the images.

**Comment by TokyoScarab** [comment]
@CarolynRowe Just checking in on this and getting the images changed out. Thanks!

**Comment by scanum** [comment]
@CarolynRowe Just checking in on this and getting the images changed out. Thanks!

**Comment by TokyoScarab** [comment]
@CarolynRowe Just following up on this. Thanks!

**Comment by TokyoScarab** [comment]
@CarolynRowe Just following up on this, thanks!

**Comment by TokyoScarab** [comment]
@CarolynRowe Any news?

**Comment by TokyoScarab** [comment]
@CarolynRowe Just following up on this, thanks!
